cloud-native-app-developer/reading-list-front-end/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="http://localhost:5173/config.js"></script>
```

Comment:
Will this work after deploying to Choreo?
I think it should be `<script src="/config.js"></script>`